packages-hybrid/hybrid-mobile-actions/scripts/exportModule.ts

```diff
@@ -0,0 +1,159 @@
+/* eslint no-console: 0 */
+import { promises as fs, existsSync, createWriteStream, mkdirSync } from "fs";
+import { join } from "path";
+import { get } from "http";
+import { decompress } from "targz";
+import { spawnSync } from "child_process";
+
+const cwd = process.cwd();
+const buildPath = join(cwd, "dist/build/");
+
+let mendixVersion = "";
+let moduleVersion = "";
+let projectPath = "";
+let moduleName = "";
+let repoMxBuildUrl = "";
+const monoPath = "/Library/Frameworks/Mono.framework/Versions/Current/Commands";
+// For local tests of unreleased mx versions: set useLocaleUtils true, and path to Mendix folder
+const useLocaleUtils = false;
+const localUtilPath = "/Volumes/[C] Windows10Enterprise/Program Files/Mendix/";
+
+main().catch(console.error);
+
+async function main(): Promise<void> {
+    /* eslint-disable-next-line */
+    const packageContent: any = require(join(cwd, "package.json"));
+    moduleVersion = packageContent.version;
+    if (
+        !packageContent.config ||
+        !packageContent.config.projectName ||
+        !packageContent.config.moduleName ||
+        !packageContent.config.testProjects ||
+        !packageContent.config.testProjects.length ||
+        !packageContent.config.testProjects[0].mendixVersion ||
+        !packageContent.config.testProjects[0].path
+    ) {
+        console.error("No module export configuration found in package.json");
+        return;
+    }
+    projectPath = join(packageContent.config.testProjects[0].path, packageContent.config.projectName);
```

Comment:
Maybe develop something that iterate in all testProjects and extract this script in the root folder